### ISSUE TITLE
[TS File Not Found Error] ENOENT: no such file or directory, open '/node_modules/@material/ripple/index.ts'

### ISSUE THREAD

**Issue by Pramothini-Sekar** [jamesmfriedman/rmwc#779]
- What RMWC Version are you using [major.minor.patch]: 7.0.3

- Name your build system [Webpack, Rollup...]: ReactJS (Yarn/NPM)

- Describe the bug with as much detail as possible:

I am working on a ReactJS Web app and wanted to utilize the ChipSet/Chip component provided by RMWC. Steps I followed to get this working: 

1. `yarn install @rmwc/chip`
2. Added `import '@rmwc/chip/styles';` to the file where I use ChipSet/Chip
3. Used `import { ChipSet, Chip } from '@rmwc/chip';`

I am able to see and use the components as expected. However, I am getting the below error for multiple files even though I am not using the TypeScript version. Unsure if I missed any steps for RMWC package usage.

> Failed to parse source map from '/node_modules/@material/ripple/index.ts' file: Error: ENOENT: no such file or directory, open '/node_modules/@material/ripple/index.ts'

- What happened, and what was supposed to happen:

Expected Output: For the package to compile with no errors

Actual Output:

> WARNING in ./node_modules/@material/base/component.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from '/node_modules/@material/base/component.ts' file: Error: ENOENT: no such file or directory, open '/node_modules/@material/base/component.ts'
 @ ./node_modules/@material/ripple/component.js 24:0-56 159:2-14
 @ ./node_modules/@material/ripple/index.js 25:0-28 25:0-28
 @ ./node_modules/@rmwc/ripple/next/foundation.js 5:0-61 19:17-36 21:17-42
 @ ./node_modules/@rmwc/ripple/next/index.js 6:0-51 66:11-30
 @ ./node_modules/@rmwc/chip/next/index.js 3:0-42 9:18-28
 @ ./src/components/Recommendation.js 10:0-43 103:41-48 105:44-48 120:43-47 135:43-47
 @ ./src/App.js 6:0-60 16:35-49
 @ ./src/index.js 7:0-24 10:33-36

> WARNING in ./node_modules/@material/base/foundation.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from '/node_modules/@material/base/foundation.ts' file: Error: ENOENT: no such file or directory, open '/node_modules/@material/base/foundation.ts'
 @ ./node_modules/@material/ripple/foundation.js 24:0-58 642:2-15
 @ ./node_modules/@material/ripple/index.js 27:0-29 27:0-29
 @ ./node_modules/@rmwc/ripple/next/foundation.js 5:0-61 19:17-36 21:17-42
 @ ./node_modules/@rmwc/ripple/next/index.js 6:0-51 66:11-30
 @ ./node_modules/@rmwc/chip/next/index.js 3:0-42 9:18-28
 @ ./src/components/Recommendation.js 10:0-43 103:41-48 105:44-48 120:43-47 135:43-47
 @ ./src/App.js 6:0-60 16:35-49
 @ ./src/index.js 7:0-24 10:33-36

> WARNING in ./node_modules/@material/chips/chip-set/component.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from '/node_modules/@material/chips/chip-set/component.ts' file: Error: ENOENT: no such file or directory, open '/node_modules/@material/chips/chip-set/component.ts'
 @ ./node_modules/@material/chips/chip-set/index.js 23:0-28 23:0-28
 @ ./node_modules/@material/chips/index.js 24:0-33 24:0-33
 @ ./node_modules/@rmwc/chip/next/foundation.js 3:0-52 19:17-34
 @ ./node_modules/@rmwc/chip/next/index.js 5:0-49 24:11-28
 @ ./src/components/Recommendation.js 10:0-43 103:41-48 105:44-48 120:43-47 135:43-47
 @ ./src/App.js 6:0-60 16:35-49
 @ ./src/index.js 7:0-24 10:33-36

> WARNING in ./node_modules/@material/chips/chip-set/constants.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from '/node_modules/@material/chips/chip-set/constants.ts' file: Error: ENOENT: no such file or directory, open '/node_modules/@material/chips/chip-set/constants.ts'
 @ ./node_modules/@material/chips/chip-set/index.js 25:0-89 25:0-89 25:0-89
 @ ./node_modules/@material/chips/index.js 24:0-33 24:0-33
 @ ./node_modules/@rmwc/chip/next/foundation.js 3:0-52 19:17-34
 @ ./node_modules/@rmwc/chip/next/index.js 5:0-49 24:11-28
 @ ./src/components/Recommendation.js 10:0-43 103:41-48 105:44-48 120:43-47 135:43-47
 @ ./src/App.js 6:0-60 16:35-49
 @ ./src/index.js 7:0-24 10:33-36

> WARNING in ./node_modules/@material/chips/chip-set/foundation.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from '/node_modules/@material/chips/chip-set/foundation.ts' file: Error: ENOENT: no such file or directory, open '/node_modules/@material/chips/chip-set/foundation.ts'
 @ ./node_modules/@material/chips/chip-set/index.js 24:0-29 24:0-29
 @ ./node_modules/@material/chips/index.js 24:0-33 24:0-33
 @ ./node_modules/@rmwc/chip/next/foundation.js 3:0-52 19:17-34
 @ ./node_modules/@rmwc/chip/next/index.js 5:0-49 24:11-28
 @ ./src/components/Recommendation.js 10:0-43 103:41-48 105:44-48 120:43-47 135:43-47
 @ ./src/App.js 6:0-60 16:35-49
 @ ./src/index.js 7:0-24 10:33-36

> WARNING in ./node_modules/@material/chips/chip-set/index.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from '/node_modules/@material/chips/chip-set/index.ts' file: Error: ENOENT: no such file or directory, open '/node_modules/@material/chips/chip-set/index.ts'
 @ ./node_modules/@material/chips/index.js 24:0-33 24:0-33
 @ ./node_modules/@rmwc/chip/next/foundation.js 3:0-52 19:17-34
 @ ./node_modules/@rmwc/chip/next/index.js 5:0-49 24:11-28
 @ ./src/components/Recommendation.js 10:0-43 103:41-48 105:44-48 120:43-47 135:43-47
 @ ./src/App.js 6:0-60 16:35-49
 @ ./src/index.js 7:0-24 10:33-36

> WARNING in ./node_modules/@material/chips/chip/component.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from '/node_modules/@material/chips/chip/component.ts' file: Error: ENOENT: no such file or directory, open '/node_modules/@material/chips/chip/component.ts'
 @ ./node_modules/@material/chips/chip/index.js 23:0-28 23:0-28
 @ ./node_modules/@material/chips/index.js 23:0-29 23:0-29
 @ ./node_modules/@rmwc/chip/next/foundation.js 3:0-52 19:17-34
 @ ./node_modules/@rmwc/chip/next/index.js 5:0-49 24:11-28
 @ ./src/components/Recommendation.js 10:0-43 103:41-48 105:44-48 120:43-47 135:43-47
 @ ./src/App.js 6:0-60 16:35-49
 @ ./src/index.js 7:0-24 10:33-36

> WARNING in ./node_modules/@material/chips/chip/constants.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from '/node_modules/@material/chips/chip/constants.ts' file: Error: ENOENT: no such file or directory, open '/node_modules/@material/chips/chip/constants.ts'
 @ ./node_modules/@material/chips/chip/index.js 25:0-83 25:0-83 25:0-83
 @ ./node_modules/@material/chips/index.js 23:0-29 23:0-29
 @ ./node_modules/@rmwc/chip/next/foundation.js 3:0-52 19:17-34
 @ ./node_modules/@rmwc/chip/next/index.js 5:0-49 24:11-28
 @ ./src/components/Recommendation.js 10:0-43 103:41-48 105:44-48 120:43-47 135:43-47
 @ ./src/App.js 6:0-60 16:35-49
 @ ./src/index.js 7:0-24 10:33-36

> WARNING in ./node_modules/@material/chips/chip/foundation.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from '/node_modules/@material/chips/chip/foundation.ts' file: Error: ENOENT: no such file or directory, open '/node_modules/@material/chips/chip/foundation.ts'
 @ ./node_modules/@material/chips/chip/index.js 24:0-29 24:0-29
 @ ./node_modules/@material/chips/index.js 23:0-29 23:0-29
 @ ./node_modules/@rmwc/chip/next/foundation.js 3:0-52 19:17-34
 @ ./node_modules/@rmwc/chip/next/index.js 5:0-49 24:11-28
 @ ./src/components/Recommendation.js 10:0-43 103:41-48 105:44-48 120:43-47 135:43-47
 @ ./src/App.js 6:0-60 16:35-49
 @ ./src/index.js 7:0-24 10:33-36

> WARNING in ./node_modules/@material/chips/chip/index.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from '/node_modules/@material/chips/chip/index.ts' file: Error: ENOENT: no such file or directory, open '/node_modules/@material/chips/chip/index.ts'
 @ ./node_modules/@material/chips/index.js 23:0-29 23:0-29
 @ ./node_modules/@rmwc/chip/next/foundation.js 3:0-52 19:17-34
 @ ./node_modules/@rmwc/chip/next/index.js 5:0-49 24:11-28
 @ ./src/components/Recommendation.js 10:0-43 103:41-48 105:44-48 120:43-47 135:43-47
 @ ./src/App.js 6:0-60 16:35-49
 @ ./src/index.js 7:0-24 10:33-36

> WARNING in ./node_modules/@material/chips/index.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from '/node_modules/@material/chips/index.ts' file: Error: ENOENT: no such file or directory, open '/node_modules/@material/chips/index.ts'
 @ ./node_modules/@rmwc/chip/next/foundation.js 3:0-52 19:17-34
 @ ./node_modules/@rmwc/chip/next/index.js 5:0-49 24:11-28
 @ ./src/components/Recommendation.js 10:0-43 103:41-48 105:44-48 120:43-47 135:43-47
 @ ./src/App.js 6:0-60 16:35-49
 @ ./src/index.js 7:0-24 10:33-36

> WARNING in ./node_modules/@material/dom/announce.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from '/node_modules/@material/dom/announce.ts' file: Error: ENOENT: no such file or directory, open '/node_modules/@material/dom/announce.ts'
 @ ./node_modules/@material/chips/chip-set/component.js 25:0-50 141:8-16
 @ ./node_modules/@material/chips/chip-set/index.js 23:0-28 23:0-28
 @ ./node_modules/@material/chips/index.js 24:0-33 24:0-33
 @ ./node_modules/@rmwc/chip/next/foundation.js 3:0-52 19:17-34
 @ ./node_modules/@rmwc/chip/next/index.js 5:0-49 24:11-28
 @ ./src/components/Recommendation.js 10:0-43 103:41-48 105:44-48 120:43-47 135:43-47
 @ ./src/App.js 6:0-60 16:35-49
 @ ./src/index.js 7:0-24 10:33-36

> WARNING in ./node_modules/@material/dom/events.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from '/node_modules/@material/dom/events.ts' file: Error: ENOENT: no such file or directory, open '/node_modules/@material/dom/events.ts'
 @ ./node_modules/@material/ripple/component.js 25:0-52 73:78-90 76:68-80 97:75-87 100:65-77
 @ ./node_modules/@material/ripple/index.js 25:0-28 25:0-28
 @ ./node_modules/@rmwc/ripple/next/foundation.js 5:0-61 19:17-36 21:17-42
 @ ./node_modules/@rmwc/ripple/next/index.js 6:0-51 66:11-30
 @ ./node_modules/@rmwc/chip/next/index.js 3:0-42 9:18-28
 @ ./src/components/Recommendation.js 10:0-43 103:41-48 105:44-48 120:43-47 135:43-47
 @ ./src/App.js 6:0-60 16:35-49
 @ ./src/index.js 7:0-24 10:33-36

> WARNING in ./node_modules/@material/dom/ponyfill.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from '/node_modules/@material/dom/ponyfill.ts' file: Error: ENOENT: no such file or directory, open '/node_modules/@material/dom/ponyfill.ts'
 @ ./node_modules/@material/ripple/component.js 26:0-49 88:15-22
 @ ./node_modules/@material/ripple/index.js 25:0-28 25:0-28
 @ ./node_modules/@rmwc/ripple/next/foundation.js 5:0-61 19:17-36 21:17-42
 @ ./node_modules/@rmwc/ripple/next/index.js 6:0-51 66:11-30
 @ ./node_modules/@rmwc/chip/next/index.js 3:0-42 9:18-28
 @ ./src/components/Recommendation.js 10:0-43 103:41-48 105:44-48 120:43-47 135:43-47
 @ ./src/App.js 6:0-60 16:35-49
 @ ./src/index.js 7:0-24 10:33-36

> WARNING in ./node_modules/@material/ripple/component.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from '/node_modules/@material/ripple/component.ts' file: Error: ENOENT: no such file or directory, open '/node_modules/@material/ripple/component.ts'
 @ ./node_modules/@material/ripple/index.js 25:0-28 25:0-28
 @ ./node_modules/@rmwc/ripple/next/foundation.js 5:0-61 19:17-36 21:17-42
 @ ./node_modules/@rmwc/ripple/next/index.js 6:0-51 66:11-30
 @ ./node_modules/@rmwc/chip/next/index.js 3:0-42 9:18-28
 @ ./src/components/Recommendation.js 10:0-43 103:41-48 105:44-48 120:43-47 135:43-47
 @ ./src/App.js 6:0-60 16:35-49
 @ ./src/index.js 7:0-24 10:33-36

> WARNING in ./node_modules/@material/ripple/constants.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from '/node_modules/@material/ripple/constants.ts' file: Error: ENOENT: no such file or directory, open '/node_modules/@material/ripple/constants.ts'
 @ ./node_modules/@material/ripple/index.js 26:0-28 26:0-28
 @ ./node_modules/@rmwc/ripple/next/foundation.js 5:0-61 19:17-36 21:17-42
 @ ./node_modules/@rmwc/ripple/next/index.js 6:0-51 66:11-30
 @ ./node_modules/@rmwc/chip/next/index.js 3:0-42 9:18-28
 @ ./src/components/Recommendation.js 10:0-43 103:41-48 105:44-48 120:43-47 135:43-47
 @ ./src/App.js 6:0-60 16:35-49
 @ ./src/index.js 7:0-24 10:33-36

> WARNING in ./node_modules/@material/ripple/foundation.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from '/node_modules/@material/ripple/foundation.ts' file: Error: ENOENT: no such file or directory, open '/node_modules/@material/ripple/foundation.ts'
 @ ./node_modules/@material/ripple/index.js 27:0-29 27:0-29
 @ ./node_modules/@rmwc/ripple/next/foundation.js 5:0-61 19:17-36 21:17-42
 @ ./node_modules/@rmwc/ripple/next/index.js 6:0-51 66:11-30
 @ ./node_modules/@rmwc/chip/next/index.js 3:0-42 9:18-28
 @ ./src/components/Recommendation.js 10:0-43 103:41-48 105:44-48 120:43-47 135:43-47
 @ ./src/App.js 6:0-60 16:35-49
 @ ./src/index.js 7:0-24 10:33-36

> WARNING in ./node_modules/@material/ripple/index.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from '/node_modules/@material/ripple/index.ts' file: Error: ENOENT: no such file or directory, open '/node_modules/@material/ripple/index.ts'
 @ ./node_modules/@rmwc/ripple/next/foundation.js 5:0-61 19:17-36 21:17-42
 @ ./node_modules/@rmwc/ripple/next/index.js 6:0-51 66:11-30
 @ ./node_modules/@rmwc/chip/next/index.js 3:0-42 9:18-28
 @ ./src/components/Recommendation.js 10:0-43 103:41-48 105:44-48 120:43-47 135:43-47
 @ ./src/App.js 6:0-60 16:35-49
 @ ./src/index.js 7:0-24 10:33-36

> WARNING in ./node_modules/@material/ripple/util.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from '/node_modules/@material/ripple/util.ts' file: Error: ENOENT: no such file or directory, open '/node_modules/@material/ripple/util.ts'
 @ ./node_modules/@material/ripple/index.js 23:0-31 24:0-16
 @ ./node_modules/@rmwc/ripple/next/foundation.js 5:0-61 19:17-36 21:17-42
 @ ./node_modules/@rmwc/ripple/next/index.js 6:0-51 66:11-30
 @ ./node_modules/@rmwc/chip/next/index.js 3:0-42 9:18-28
 @ ./src/components/Recommendation.js 10:0-43 103:41-48 105:44-48 120:43-47 135:43-47
 @ ./src/App.js 6:0-60 16:35-49
 @ ./src/index.js 7:0-24 10:33-36

**Comment by thaestrup** [comment]
Are you using another version of Webpack than the one provided by RMWC. These warnings usually appears when using the newer Webpack 5. They exists because our upstream dependency to @material does not provide the source in the distribution. We are working on an upgrade of a number of dependencies including Webpack and in that upgrade we have disabled these warnings. The proper fix would be for the upstream dependency to provide the source code in the distribution so maybe these will come when we upgrade to newer versions of @material. When we are fully upgraded to the newest version of @material we will report the issue upstream if it persists.
TLDR; Just ignore them for now.

**Comment by Pramothini-Sekar** [comment]
Thanks a ton, Thomas!

Yes, I'm using the latest version for everything.

As you mentioned, I shall just ignore the error messages for now. 👍🏼
